pygmt/__init__.py

```diff
@@ -47,7 +47,7 @@
 
 # Get semantic version through setuptools-scm
 __version__ = f'v{get_distribution("pygmt").version}'  # e.g. v0.1.2.dev3+g0ab3cd78
-__commit__ = __version__.split("+g")[-1]  # 0ab3cd78
+__commit__ = __version__.split("+g")[-1] if "+g" in __version__ else ""  # 0ab3cd78
```

Comment:
If we set the commit hash as an empty string, then won't the link mentioned at #1171 just become https://github.com/GenericMappingTools/pygmt/commit/ which redirects to the master branch?

Reply:
If `__commit__` is empty, the "Revision xxxx" won't show.

See https://github.com/readthedocs/sphinx_rtd_theme/blob/aa28eecc3a0ea7a28efa0880bfcec6abaabdb1fa/sphinx_rtd_theme/footer.html#L32

Reply:
@weiji14 You're right here, so I have to set the variable `commit_link` in `conf.py` to an empty string. See the changes in 6b42847bcefad3ded0356cd3238cf323eeeacee6.

Preview for a local build when `__version__ == "v0.3.1"`

![image](https://user-images.githubusercontent.com/3974108/113535006-3c113a00-95a0-11eb-9377-b891169b23a0.png)